addons/website/views/website_templates.xml

```diff
@@ -1925,6 +1925,8 @@
          t-att-data-use_text_align="use_text_align"
          t-att-data-res-model="request.env.user.has_group('website.group_website_publisher') and _record._name"
          t-att-data-res-id="request.env.user.has_group('website.group_website_publisher') and _record.id"
+         t-att-data-bg-color-style="_cp.get('background_color_style')"
+         t-att-data-bg-color-class="_cp.get('background_color_class')"
```

Comment:
Having to do that is not ideal, there must be a better fix. I'm not sure to understand the problem from your commit message, you should be able to find which color is currently used on editor start without having to print it like here.